src/app/(views)/enrolled-courses/[courseId]/modules/page.tsx

```diff
@@ -5,40 +5,142 @@ import Link from 'next/link'
 import { useParams } from 'next/navigation'
 import useGetCourses from '@/hooks/useGetCourses'
 import { Loading } from '@/components'
-export default function Modules() {
+import { 
+  CCard,
+  CCardBody,
+  CCardTitle,
+  CCardText,
+  CButton,
+  CRow,
+  CCol,
+  CBadge,
+  CDropdown,
+  CDropdownItem,
+  CDropdownMenu,
+  CDropdownToggle,
+  CTable,
+  CTableBody,
+  CTableDataCell,
+  CTableHead,
+  CTableHeaderCell,
+  CTableRow,
+} from '@coreui/react-pro'
+import CIcon from '@coreui/icons-react'
+import {
+  cilPencil,
+  cilTrash,
+  cilPlus,
+  cilPenAlt,
+  cilPeople,
+  cilFile,
+  cilNotes,
+  cilVideo,
+  cilImage,
+} from '@coreui/icons'
+
+export default function Course() {
   const params = useParams()
   const { courseId } = params as { courseId: string }
-
   const { courses, fetchingCourses } = useGetCourses({ courseId })
+  const course = courses[0]
 
-  if (fetchingCourses) {
-    return <Loading />
+  if (fetchingCourses || !course) {
```

Comment:
If there's no course `!course `, why are displaying a loading component?
If we finished loading and there's no course, we should display an error message.